mrp_unbuild_advanced/models/mrp_unbuild.py

```diff
@@ -68,6 +63,8 @@ def action_validate(self):
                     "Shift dates must be filled before"
                     " unbuild action for %s is performed!"
                 ) % unbuild.name)
+            if self.unbuild_date > fields.Datetime.now():
```

Comment:
I don't understand why this check is placed here. According to requirements (check if date is still in the future when unbuild is marked as done), check should be made when "Unbuild button" is clicked:

https://github.com/odoo/odoo/blob/0d5b386c32db609d9a1d07ffe515d77fc83c3786/addons/mrp/models/mrp_unbuild.py#L137

Could you check?

Reply:
You're using `self`, but `action_validate()` is multi, please fix it

Reply:
Yeah, mb. Fixed